.circleci/config.yml

```diff
@@ -135,8 +141,7 @@ jobs:
 
       - restore_cache:
           keys:
-            - -pkg2-v20220303-{{ checksum "Makefile.envs" }}
-            - -pkg2-v20220303
+            - -pkg3-v20220915-{{ checksum "build_hash.txt" }}-<< parameters.packages >>
```

Comment:
Note that I used seperate cache keys for each incremental build jobs.
I wasn't able to find a way to mix `persist_to_workspace` and `save_cache` to make a single cache work...